sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIOReadImplementationCompatibility.java

```diff
@@ -137,6 +137,7 @@ Object getDefaultValue() {
         return false;
       }
     },
+    OFFSET_DEDUPLICATION(LEGACY),
```

Comment:
this looks odd to me, shouldn't there be a } brace to match the start of the enum {? seems like that before but not sure why it works

Reply:
It confused me at first as well, previously I had only worked with simple enums. In Java, enums can have constructors, methods, etc., like classes. This enum has a few methods below with the closing brace at L179 at head, L180 on this change.

Reply:
Simpler example: http://screen/83tVFLVSnSEhfyK